src/string_enums/enum_base.hpp

```diff
@@ -82,7 +83,8 @@ struct enum_base : public T
 };
 
 #define ENUM_AND_ARRAY(...) \
+	static constexpr std::size_t count = std::tuple_size<decltype(std::make_tuple(__VA_ARGS__))>::value; \
```

Comment:
So I guess the implicit template arguments didn't work?

Reply:
Yeah, for the mingw crosscompile job, it seemed to confuse the linker when it wasn't provided explicitly for some reason.